# Type: docs
# Subtype: D2
# English Summary: Add TODO comments for TransformUtil
# Chinese Summary: 为TransformUtil添加TODO注释
# Entry Number: 28

# === Git Diff Content ===

diff --git a/core/src/main/java/com/airbnb/aerosolve/core/util/TransformUtil.java b/core/src/main/java/com/airbnb/aerosolve/core/util/TransformUtil.java
index 67b0e5a..e3ff3aa 100644
--- a/core/src/main/java/com/airbnb/aerosolve/core/util/TransformUtil.java
+++ b/core/src/main/java/com/airbnb/aerosolve/core/util/TransformUtil.java
@@ -1,7 +1,9 @@
 package com.airbnb.aerosolve.core.util;
 
 /**
- * Created by christhetree on 30/04/16.
+ * Utility class for methods that are commonly used by transforms and should be shared.
+ * TODO: go through all transforms and add all common methods here.
+ * TODO: go through Util class and add all transform related / exclusive methods here.
  */
 public class TransformUtil {
     public static Double quantize(double val, double delta) {
